[2024-12-17T13:28:34.102+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-17T13:28:34.117+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: stock_information.stream_data_from_api manual__2024-12-17T13:28:31.852604+00:00 [queued]>
[2024-12-17T13:28:34.125+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: stock_information.stream_data_from_api manual__2024-12-17T13:28:31.852604+00:00 [queued]>
[2024-12-17T13:28:34.128+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2024-12-17T13:28:34.142+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): stream_data_from_api> on 2024-12-17 13:28:31.852604+00:00
[2024-12-17T13:28:34.149+0000] {standard_task_runner.py:72} INFO - Started process 216 to run task
[2024-12-17T13:28:34.153+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'stock_information', 'stream_data_from_api', 'manual__2024-12-17T13:28:31.852604+00:00', '--job-id', '36', '--raw', '--subdir', 'DAGS_FOLDER/vnstock_stream.py', '--cfg-path', '/tmp/tmpl8271i84']
[2024-12-17T13:28:34.157+0000] {standard_task_runner.py:105} INFO - Job 36: Subtask stream_data_from_api
[2024-12-17T13:28:34.201+0000] {task_command.py:467} INFO - Running <TaskInstance: stock_information.stream_data_from_api manual__2024-12-17T13:28:31.852604+00:00 [running]> on host 453ac4defedc
[2024-12-17T13:28:34.280+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='TuanDz' AIRFLOW_CTX_DAG_ID='stock_information' AIRFLOW_CTX_TASK_ID='stream_data_from_api' AIRFLOW_CTX_EXECUTION_DATE='2024-12-17T13:28:31.852604+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-17T13:28:31.852604+00:00'
[2024-12-17T13:28:34.283+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-17T13:31:05.887+0000] {local_task_job_runner.py:346} WARNING - State of this instance has been externally set to success. Terminating instance.
[2024-12-17T13:31:05.891+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2024-12-17T13:31:05.895+0000] {process_utils.py:132} INFO - Sending 15 to group 216. PIDs of all processes in the group: [216]
[2024-12-17T13:31:05.897+0000] {process_utils.py:87} INFO - Sending the signal 15 to group 216
[2024-12-17T13:31:05.900+0000] {taskinstance.py:3093} ERROR - Received SIGTERM. Terminating subprocesses.
[2024-12-17T13:31:05.925+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-17T13:31:05.953+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=216, status='terminated', exitcode=0, started='13:28:33') (216) terminated with exit code 0
